Docs/Configuration.tex

```diff
@@ -2573,6 +2573,13 @@ \subsection{Quirks Properties}\label{kernelpropsquirks}
   on all systems but those explicitly dedicated to scientific or media calculations.
   Only certain Xeon models typically benefit from the patch.
 
+\item
+  \texttt{CustomPCISerialDevice}\\
+  \textbf{Type}: \texttt{plist\ boolean}\\
+  \textbf{Failsafe}: \texttt{false}\\
+  \textbf{Requirement}: 10.4\\
```

Comment:
I am not sure what the minimum requirement is, though the lowest version makes sense to me.